OasisDEX/oasis-borrow
# Legal update [DO NOT MERGE]

# [Update Cookie Banner](https://app.shortcut.com/oazo-apps/story/5243/update-cookie-banner)


  
## Changes 👷‍♀️
added `version` to the cookie policy settings

`Update the link from the Privacy Policy to the Cookie Policy in the copy`

`Update the copy from the marketing cookies to:
"Oasis.app uses anonymised cookies from adRoll to be able to apply retargeting campaigns on advertising platforms"`

`Standard untick the cookies, and only tick them when the user clicks accept`
- updated terms of service
- updatet privacy policy
- added cookie policy
- added GDPR copy (when newsletter input is onFocus)
  
## How to test 🧪
- go to the homepage - cookie banner should pop up automatically
- check if TOS modal popped up
